notebooks/efficient-gpu-utilization.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>Do you mean <strong>with</strong> quantization? </p><p>Can you provide the percentage change in memory and inference time? it is more intuitive.</p><p>For inference time computation, can you run the cell 10 times and average the time? That makes the result more robust.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>